src/Workspaces/Core/Portable/Classification/ClassificationProviderNames.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Classification
+{
+    internal static class ClassificationProviderNames
```

Comment:
I think our pattern is like "PredefinedClassificationProviderNames".  But i coudl be wrong about that.